app/fetchers/service_plan_visibility_fetcher.rb

```diff
@@ -0,0 +1,15 @@
+module VCAP::CloudController
+  class ServicePlanVisibilityFetcher
+    class << self
+      def service_plans_visible_in_orgs?(service_plan_guids, readable_org_guids)
+        empty = ServicePlanVisibility.dataset.
+                left_join(:organizations, id: Sequel[:service_plan_visibilities][:organization_id]).
+                left_join(:service_plans, id: Sequel[:service_plan_visibilities][:service_plan_id]).
+                where { (Sequel[:service_plans][:guid] =~ service_plan_guids) & (Sequel[:organizations][:guid] =~ readable_org_guids) }.
```

Comment:
What is this line doing? What are those `=~` operators in this context?

Reply:
The `=~` operator is documented here: https://sequel.jeremyevans.net/rdoc/files/doc/dataset_filtering_rdoc.html

We felt that it was more readable than the alternative, but are open to changing it if folks are more familiar with other ways of doing it.